README.md

```diff
@@ -35,9 +35,14 @@ It can be enabled as a user systemd service:
 ```
 $ systemctl --user enable --now replay-sorcery
 ```
-
 Once it is running, just press Ctrl+Super+R to save the last 30 seconds.
 
+When the configuration file has changed, the service must be reloaded by running:
+
+```
```

Comment:
np: can you remove the empty newline here to make it consistent like the rest of the file?